app/views/technologies/index.html.erb

```diff
@@ -1 +1,9 @@
-<%= render @technologies || 'empty_list' %>
+<div class="header">
+
+</div>
+<div class="container">
+  <div class="tech-container">
+    <%= render @technologies || 'empty_list' %>
```

Comment:
First of all, use parentheses in cases like this, it's unclear what's precedence.
``` ruby
2.5.1 :001 > def m(*args); p args; end
 => :m
2.5.1 :002 > m nil || "nothing"
["nothing"]
 => ["nothing"]
```

So when `@technologies` is nil it calls `render('empty_list')`. I don't see such template in this PR, so something is broken here.